DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -67,39 +66,65 @@ function Set-TargetResource
         [string]
         $Params,
         [ValidateNotNullOrEmpty()]
-        [string]
+        [string[]]
         $Version,
         [string]
         $Source,
-        [String]
+        [string]
         $chocoParams,
         [bool]
         $AutoUpgrade = $false
     )
     Write-Verbose -Message 'Start Set-TargetResource'
+    $env:Path = [Environment]::GetEnvironmentVariable('Path','Machine')
+    Write-Verbose -Message "Path variables: $env:Path"

```

Comment:
Can you explain why this is here? It doesn't appear to be used for anything other than to output a Verbose message.